lib/types/hostnametrie.go

```diff
@@ -0,0 +1,162 @@
+package types
```

Comment:
Don't forget to add the copyright header to this file and `hostnametrie_test.go`.

Reply:
fixed in 2f729eca